src/orders/utils/data.ts

```diff
@@ -208,3 +210,15 @@ export function mergeRepeatedOrderLines(
     return prev;
   }, Array<OrderDetails_order_fulfillments_lines>());
 }
+
+export const getVariantSearchAddress = (order: OrderDetails_order) => {
+  if (!!order.shippingAddress) {
```

Comment:
`!!` are not needed inside statements as they are automatically cast to booleans